lib/vmdb/settings/walker.rb

```diff
@@ -1,8 +1,8 @@
 module Vmdb
-  class Settings
-    module Walker
-      PASSWORD_FIELDS = %i(bind_pwd password amazon_secret ssh_key_data ssh_key_unlock become_password vault_password security_token).to_set.freeze
+  module SettingsWalker
```

Comment:
> But left out the reasons why because most people complain that I write too much... but since you asked for it, here you go (sucker)!

@NickLaMuro 🤣